client/src/redux/users/usersactions.js

```diff
@@ -0,0 +1,14 @@
+import types from '../types';
```

Comment:
this file can be deleted. I should have deleted the billsactions.js file as well but I haven't gotten around to it. This serves the same purpose as the actions.js file.